cmd/flags.go

```diff
@@ -126,8 +127,19 @@ func addInputFlag(cmd *cobra.Command) {
 
 // Lets this command specify an Attestation Server Address.
 func addAsAddressFlag(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringVar(&asAddress, "asAddr", "https://confidentialcomputing.googleapis.com",
-		"Attestation Service address")
+	cmd.PersistentFlags().StringVar(&asAddress, "verifier-endpoint", "https://confidentialcomputing.googleapis.com",
+		"the attestation verifier endpoint used to retrieve an attestation claims token")
+}
+
+// Lets this command enable Cloud logging
+func addCloudLoggingFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool("cloud-log", false, "cloud logging switch")
+}
+
+// Lets this command specify custom audience field of the attestation token
+func addAudienceFlag(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&audience, "audience", "https://api.example.com",
+		"the audience field in the claims token")
```

Comment:
the audience field in the claims token. Cannot be sts.googleapis.com.

Reply:
In theory, if the malicious actor really want this field to be sts.googleapis.com. He can use the api to fetch it anyway. Shall I put a note on "Cannot be sts.googleapis.com" or else? What do you recommend?

Reply:
The service rejects the https://sts.googleapis.com audience

Reply:
Thanks for the explanation! Confirmed. I have another question. Was it a bug that I could get "https://sts.googleapis.com/" before? Or is there a recent change for the service?

Reply:
I think the trailing `/`